src/accelerate/inference.py

```diff
@@ -132,6 +138,9 @@ def prepare_pippy(
         num_chunks (`int`):
             The number of different stages the Pipeline will have. By default it will assign one chunk per GPU, but
             this can be tuned and played with. In general one should have num_chunks >= num_gpus.
+        send_output_to_cpu (`bool`, defaults to `True`):
+            Whether to broadcast the model output across processes and send it to the CPU afterwards or to leave the
+            model output on the last device.
```

Comment:
Is it the default case that Accelerate would move result from GPU to CPU for the user? (Regardless of parallelism)
I am just thinking that some commands like `print(output)` (if user needs it) would also move the tensor from GPU to CPU, but that keeps the move more "as needed".
I am just referring to the GPU->CPU part (not the broadcast part), although both of them may have performance implications.

Reply:
Yes, this is aligned with how we do things in transformers so we're keeping this behavior. cc @SunMarc

Reply:
However to keep it barebones, we won't move to the CPU, just keep everything on the GPU for the output